src/main/java/com/sprint/mission/discodeit/service/basic/BasicUserService.java

```diff
@@ -0,0 +1,111 @@
+package com.sprint.mission.discodeit.service.basic;
+
+import com.sprint.mission.discodeit.dto.MessageDto.BinaryContentDto;
+import com.sprint.mission.discodeit.dto.UserDto.UserCreateDto;
+import com.sprint.mission.discodeit.dto.UserDto.UserDto;
+import com.sprint.mission.discodeit.dto.UserDto.UserStatusDto;
+import com.sprint.mission.discodeit.entity.BinaryContent;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.repository.BinaryContentRepository;
+import com.sprint.mission.discodeit.repository.UserRepository;
+import com.sprint.mission.discodeit.repository.UserStatusRepository;
+import com.sprint.mission.discodeit.service.BinaryContentService;
+import com.sprint.mission.discodeit.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@Service
+public class BasicUserService implements UserService, BinaryContentService {
```

Comment:
BasicUserService는 UserService만 implements 받아야할 것 같아요.  
BasicUserService가 UserService, BinaryContentService 모두 implements 받는다면 두 Service의 책임을 동시에 수행하는 것이라서요.
```suggestion
public class BasicUserService implements UserService {
```